packages/@sanity/form-builder/src/sanity/DefaultAssetSource/utils/mimeType.ts

```diff
@@ -0,0 +1,132 @@
+interface MimeTypeInfo {
+  title: string
+}
+
+const MIME_TYPES: Record<string, MimeTypeInfo> = {
+  // Images
+  'image/bmp': {
+    title: 'Bitmap',
+  },
+  'image/jpeg': {
+    title: 'JPEG',
+  },
+  'image/gif': {
+    title: 'GIF',
+  },
+  'image/vnd.microsoft.icon': {
+    title: 'Icon',
+  },
+  'image/png': {
+    title: 'PNG',
+  },
+  'image/svg+xml': {
+    title: 'SVG',
+  },
+  'image/webp': {
+    title: 'WEBP',
+  },
+  'image/tiff': {
+    title: 'TIFF',
+  },
+  'image/heic': {
+    title: 'HEIC',
+  },
+  // Audio
+  'audio/midi': {
+    title: 'MIDI',
+  },
+  'audio/midi-x': {
+    title: 'MIDI',
+  },
+  'audio/mpeg': {
+    title: 'MP3 Audio',
+  },
+  'audio/ogg': {
+    title: 'OGG Audio',
+  },
+  'audio/wav': {
+    title: 'WAW Audio',
+  },
+  'audio/webm': {
+    title: 'WEBM Audio',
+  },
+  'audio/aac': {
+    title: 'AAC Audio',
+  },
+  // Video
+  'video/x-msvideo': {
+    title: 'AVI',
+  },
+  'video/mp4': {
+    title: 'MP4',
+  },
+  'video/mpeg': {
+    title: 'MPEG',
+  },
+  'video/ogg': {
+    title: 'OGG',
+  },
+  'video/webm': {
+    title: 'WEBM',
+  },
+  'video/quicktime': {
+    title: 'QuickTime',
+  },
+  // Documents
+  'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet': {
+    title: 'Microsoft Excel',
+  },
+  'application/vnd.ms-excel': {
+    title: 'Microsoft Excel',
+  },
+  'text/plain': {
+    title: 'Text',
+  },
+  'text/javascript': {
+    title: 'JavaScript',
+  },
+  'text/markdown': {
+    title: 'Markdown',
+  },
+  'text/csv': {
+    title: 'text/csv',
```

Comment:
```suggestion
    title: 'CSV File',
```

Reply:
We don't use the File suffix in any of the other files, and I think CSV/Comma Separated Values reads just fine, so will skip this.